### ISSUE TITLE
User Content Accessiblity Issue Collection

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4085]
This issue is a placeholder for anything accessibility concerns that are with user content rather than under the control of the team for consideration in Phase 2 of the accessibility project. 

Add items as they arise into the comments below.

**Comment by StephDriver** [comment]
- [ ] News Articles headings that rely on image content to convey meaning.
for example from BirkbeckCTP/hourglass#324

## Images within snippets e.g. on news pages, journal pages with a link to that item
It could be argued that most of these are decorative, for example in the list of journals there is the journal cover as a thumbnail next to the name of that journal and a link to the journal page.  The cover should be described on the journal page itself, but on following a few of these links, the journal cover is not then on the landing page.  So there is an argument for needing to describe the journal cover image.

When it comes to new and announcement items, where the headline is more variable, this is not always the case.  Sometimes the thumbnail is of the banner image, and again this would be described on the linked page.  But sometimes the thumbnail contains vital information, for example: 

![Image](https://github.com/BirkbeckCTP/hourglass/assets/5330770/79ca73fe-0fa8-4095-9ba5-9798fcd3db2c)

The image of "The Comics Grid, Journal of comics scholarship" is set to be ignored. But the title refers to volumes, without the name of the journal, and thus that image contains important information for the context of the link, which is not being conveyed other than visually.

**Comment by StephDriver** [comment]
Include functionality to assist users with specifying language for words/phrases when putting text in via TinyMCE. as discussed in 
- BirkbeckCTP/hourglass#340

**Comment by StephDriver** [comment]
Support users to better use MathJax, as some are thought to be using images of equations instead. This is likely to involve both authors/editors and subcontracted typesetting to get it right.

**Comment by StephDriver** [comment]
Support users to design accessible data-tables, such that they have column and row headers properly specified in the JATS. (see #4055)

**Comment by StephDriver** [comment]
Include warnings/training for editors on A11y issues of having long lists of articles on the homepage.  see #4242

**Comment by StephDriver** [comment]
Templates for authors - preferably including free softwares and not just MS, optimised for the markup we need for typesetting (lang markup, styles etc) but also able to be *easily and quickly* customised for each journal - both branding- and expected content-wise.

**Comment by StephDriver** [comment]
cross references to other sections (e.g. a 'see section 4') are not consistently marked up as internal links, even within the same article, some will be marked as links, some will be plain text. (see #4245)

**Comment by StephDriver** [comment]
structure of tables an lists within articles.
